### ISSUE TITLE
occur lookup <customized bcrypt hasher> on passlib.handlers.bcrypt failed when I use RedisCache

### ISSUE THREAD

**Issue by neeker** [langflow-ai/langflow#3253]
### Bug Description

occur lookup <customized bcrypt hasher> on passlib.handlers.bcrypt failed when I use RedisCache：

<img width="1194" alt="image" src="https://github.com/user-attachments/assets/51d367df-05d4-420f-bc3c-965b9edae2aa">

### Reproduction

class RedisCache(AsyncBaseCacheService, Generic[LockType]):  # type: ignore

   ...

    async def set(self, key, value, lock=None):
        """
        Add an item to the cache.

        Args:
            key: The key of the item.
            value: The value to cache.
        """
        try:
            if pickled := pickle.dumps(value):
                result = self._client.setex(str(key), self.expiration_time, pickled)
                if not result:
                    raise ValueError("RedisCache could not set the value.")
        except TypeError as exc:
            raise TypeError("RedisCache only accepts values that can be pickled. ") from exc

### Expected behavior

when I use RedisCache and run build buidlFlow, I got error of pickle.

### Operating System

MacOS

### Langflow Version

1.0.14

### Python Version

3.12

### Screenshot

<img width="1194" alt="image" src="https://github.com/user-attachments/assets/51d367df-05d4-420f-bc3c-965b9edae2aa">

### Flow File

[test.json](https://github.com/user-attachments/files/16557207/test.json)

**Comment by edwinjosechittilappilly** [comment]
Thank you for reporting the pickling issue with RedisCache in Langflow. We appreciate your contribution to improving the tool.

**The issue has been resolved in Langflow version 1.0.17.**
Please make sure you're using the latest version.

I've confirmed that RedisCache is working as expected with the latest Langflow version. Please let me know if you encounter any issues even after upgrading.

**To use RedisCache, please set the following environment variables:**

```
LANGFLOW_CACHE_TYPE=redis
LANGFLOW_REDIS_HOST=<your_redis_host>  # Replace with your Redis host
LANGFLOW_REDIS_PORT=<your_redis_port>    # Replace with your Redis port
LANGFLOW_REDIS_DB=<your_redis_db>      # Replace with your Redis database number
LANGFLOW_REDIS_CACHE_EXPIRE=<your_redis_cache_expiration> # Sets cache expiration in seconds
```

**Replace the placeholders with the appropriate values for your Redis deployment.**

**Comment by neeker** [comment]
When enable streaming and redis, You can get error:

> RedisCache only accepts values that can be pickled.
